index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio-Ishpaul Singh</title>
+</head>
+<body>
+    <header>
+        <!--The nav bar-->
+        <nav>
+            <!--logo-->
+            <div class="logo">
+                <h1>ISHPAUL SINGH</h1>
+            </div>
+            <!--menu icon-->
+            <div class="toggle-menu">
+                <a href="#"><img src="assets/icons/menu-icon.svg" alt="menu"></a>
+            </div>
+        </nav>
+
+    </header>
+        <!--INTRODUCTION SECTION-->
+        <section class="intro-section">
+            <!--HEADLINE-->
+            <h1 class="headline">
+                Hii!<br>
+                I'm Ishpaul Singh<br>
+                Glad to see you!
+            </h1>
+            <!--description-->
+            <p class="description">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <!--social-handles-->
+            <div class="social-handles">
+                <p>Let's Connect</p>
+                <ul class="social-icons">
+                    <li class="github"><a href="#"><img src="/assets/icons/github.svg" alt="github"></a></li>
+                    <li class="linkedin"><a href="#"><img src="/assets/icons/linkedin.svg" alt="linkedin"></a></li>
+                    <li class="twitter"><a href="#"><img src="/assets/icons/twitter.svg" alt="twitter"></a></li>
+                    <li class="instagram"><a href="#"><img src="/assets/icons/instagram.svg" alt="instagram"></a></li>
+                </ul>
```

Comment:
- [OPTIONAL] I would recommend you to use [Fontawesome ](https://fontawesome.com/)for you to configure icons, you will find very nice-looking icons for your project. Ths make your project look more uniform.